tests/components/withings/test_common.py

```diff
@@ -121,21 +114,21 @@ def test_print_service() -> None:
     hass: HomeAssistant, data_manager: WithingsDataManager,
 ) -> None:
     """Test method."""
-    await async_process_ha_core_config(
-        hass=hass, config={"time_zone": "America/Los_Angeles"}
-    )
-
-    update_start_time = dt.now()
-    await data_manager.update_sleep()
-
-    call_args = data_manager.api.sleep_get.call_args_list[0][1]
-    startdate = call_args.get("startdate")
-    enddate = call_args.get("enddate")
-
-    assert startdate.tzname() == "PST"
-
-    assert enddate.tzname() == "PST"
-    assert startdate.tzname() == "PST"
-    assert update_start_time < enddate
-    assert enddate < update_start_time + timedelta(seconds=1)
-    assert enddate > startdate
+    with patch(
```

Comment:
If we move this to a pytest fixture function and yield within the patch context manager we can outdent the test body below again.

Reply:
Sure! Done.